examples/webgpu_postprocessing_ssr.html

```diff
@@ -113,19 +119,24 @@
 			scenePass.setMRT( mrt( {
 				output: output,
 				normal: directionToColor( normalView ),
-				metalness: metalness
+				metalness: metalness,
+				roughness: roughness,
```

Comment:
@sunag Are you aware if we could to this:
```js
metalnessRoughness: vec2( metalness, roughness )
```
And later:
```js
const metalnessRoughnessTexture = scenePass.getTexture( 'metalnessRoughness' );
const metalness = metalnessRoughnessTexture.r;
const roughness = metalnessRoughnessTexture.g;
```
That would save us one attachment.